src/main/java/com/google/sps/data/DataGraph.java

```diff
@@ -0,0 +1,125 @@
+package com.google.sps.data;
+
+// import java.util.ArrayList;
```

Comment:
Revert these comments?

Reply:
@fenghaolw even though I pushed some code to this branch right now, the code hasn't been modified to address your comments fully. We'll comment here once it's ready for your review :)

Reply:
Other than the replies to your comments for what I wasn't sure about, I think I addressed them! :)